pkg/cloudprovider/provider/kubevirt/provider_test.go

```diff
@@ -167,7 +168,12 @@ func (k kubevirtProviderSpecConf) rawProviderSpec(t *testing.T) []byte {
 					"osImage": "http://x.y.z.t/ubuntu.img",
 					{{- end }}
 					"size": "10Gi",
+					{{- if .OsImageSource }}
+					"storageClassName": "longhorn",
```

Comment:
As `storageClassName` is set to "longhorn" in the if and the else, it could go out of the if/else condition.

Reply:
there is https://github.com/kubermatic/machine-controller/pull/1470/files#diff-2e539081e4b1bd387e1af416faf12022e219faba4f706b478b0f25523315a7a4R175

it could be just one if to add `OsImageSource`  true, but the  current state doesn't break the behaviour.